Client/Frontend/Browser/LoginsHelper.swift

```diff
@@ -209,8 +218,12 @@ class LoginsHelper: TabContentScript {
             }
 
             let json = JSON(jsonObj)
-            let src = "window.__firefox__.logins.inject(\(json.stringValue()!))"
-            self.tab?.webView?.evaluateJavaScript(src, completionHandler: { (obj, err) -> Void in
+            guard let jsonString = json.stringValue() else {
+                return
+            }
+            
+            self.tab?.webView?.evaluateSafeJavascript(functionName: "window.__firefox__.logins.inject", args: [jsonString], sandboxed: false, escapeArgs: false, completion: { (obj, err) -> Void in
+                log.debug(err)
```

Comment:
Should probably only log the error if its non-nil?

Reply:
done!